### PR TITLE
chore: Add ThePyroTF2

### PR DESCRIPTION
**Pull request by ThePyroTF2**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1655"></a>

### CODE REVIEW

File: public/data/ThePyroTF2.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "url": "https://github.com/ThePyroTF2",
+        "icon": "github"
+      },
+      {
+        "name": "Twitter",
+        "url": "https://twitter.com/JustDeeevin",
+        "icon": "twitter"
+      },
+      {
+        "name": "Youtube",
```
Comment (Panquesito7) [inline public/data/ThePyroTF2.json:18]: I recommend capitalizing the `t` here. What do you think? 🙂
```suggestion
        "name": "YouTube",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Comment (ThePyroTF2) [reply public/data/ThePyroTF2.json:18]: I don't usually capitalize the `t` in YouTube but I do realize that's its official spelling. I'll make the change.

Good suggestion, thanks :)

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contributions :tada:

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

**Pull request merged**